src/client/app/reducers/formReducer.js

```diff
@@ -0,0 +1,68 @@
+import {
+    FORM_SUCCESS,
+    SUBMIT_ERROR,
+    SET_FORM_FIELDS,
+    UPDATE_FORM_FIELD,
+    SET_USER_AUTOCOMPLETE
+} from '../constants';
+
+const initialstate = {
+    errors: []
+};
+
+export default function formReducer (state = initialstate, action) {
+    switch (action.type) {
+
+    case SET_FORM_FIELDS:
+        const { fields, modelName, formType } = action.data;
+        return {
+            ...state,
```

Comment:
🤔 I think we can skip note about browser compatibility, but note about ES7 features might be nice for those of us who aren't as familiar with the patterns. 

Good deal that they're compatible! It's always such a crapshoot these days...